utils/ccloud_library.sh

```diff
@@ -356,7 +356,8 @@ function ccloud::maybe_create_and_use_cluster() {
 function ccloud::create_service_account() {
   SERVICE_NAME=$1
 
-  OUTPUT=$(ccloud service-account create $SERVICE_NAME --description $SERVICE_NAME  -o json)
+  CCLOUD_EMAIL=$(ccloud prompt -f '%u')
+  OUTPUT=$(ccloud service-account create $SERVICE_NAME --description "$EXAMPLE run by $CCLOUD_EMAIL"  -o json)
```

Comment:
The other thing we are typically missing are the resource creation dates.  Would it be possible to add a datetime stamp, either here in the service-account description, or in the environment name (which to be fair would be getting quite busy then).

Reply:
@javabrett  I can get env creation date...so if that data is useful for you, I can share the link (DM in Slack please).  But stepping back, we should think about how far we want to go in this...making up for things the product itself doesn't do yet. 😄